src/ts/api.ts

```diff
@@ -161,20 +163,36 @@ async function estimateTradeAmount({
   amount,
   baseUrl,
 }: EstimateTradeAmountQuery & ApiCall): Promise<BigNumber> {
-  const response: EstimateAmountResponse = await call(
-    `markets/${sellToken}-${buyToken}/${apiKind(kind)}/${BigNumber.from(
-      amount,
-    ).toString()}`,
```

Comment:
Since this route isn't available anymore, I wonder if we should drop this function. To me it's unclear what the output of `estimateTradeAmount` should be for buy orders without further documentation, and it looks like we only use this function for sell orders (I also checked the code of the trading bot, where it appears once).
It would require a few changes in the tests since we mock the output of this function however.

Reply:
I'm going to leave it in for now to reduce the maintenance work.